client/components/wcc-settings-form/index.js

```diff
@@ -35,7 +35,10 @@ const WCCSettingsForm = ( props ) => {
 		const label = ( currentStepLayout || {} ).action_label || __( 'Next' );
 		return (
 			<FormButtonsBar>
-				<FormButton type="button" onClick={ props.formActions.nextStep }>
+				<FormButton
+					type="button"
+					disabled={ ! currentStepLayout || 0 < props.stepErrors.length || props.form.isSaving }
+					onClick={ props.formActions.nextStep }>
```

Comment:
I wonder if we should have a separate component for these multi-step forms to reduce the risk of collateral damage to the settings and self-help views. Keeping multi-step and single-step logic clean in one component could be challenging.  We could DRY things with utility requires.

@jeffstieler - what do you think?